pydantic/dataclasses.py

```diff
@@ -106,11 +110,46 @@ def dataclass(
     kw_only: bool = False,
 ) -> Callable[[type[_T]], type[PydanticDataclass]] | type[PydanticDataclass]:
     """
-    Like the python standard lib dataclasses but enhanced with validation.
+    Returns a decorator that creates a Pydantic-enhanced dataclass, similar to the standard Python `dataclasses`,
+    but with added validation.
```

Comment:
We are not using linter, but to avoid changes later on:
```suggestion
    A decorator used to create a Pydantic-enhanced dataclass, similar to the standard Python `dataclasses`,
    but with added validation.
```
(sentences should be in imperative, words like "creates", "returns", and anything that ends with "s" as first word, the linter will complain)